Cargo.lock

```diff
@@ -163,9 +163,9 @@ dependencies = [
 
 [[package]]
 name = "anyhow"
-version = "1.0.67"
```

Comment:
are you sure about this Cargo.lock ? That's a lot of changes.

Reply:
it was a little out of date, but still lots of changes.

Is there a best practice to update it other then deleting and rebuilding

Reply:
manta-rs version does go from v0.5.8 -> v0.5.12. But please check it is correct

Reply:
I don't delete it for that reason.
When you change cargo.toml branches doesn't it update on its own ?
The other thing I do, if it can't update on its own, is to update 1 package at a time, only the things I need.
I only delete or cargo update all if I can't get it to compile any other way

Reply:
I just built it, but I think a big reason for the change is that the current Cargo.lock is actually out of date. 

manta-rs version in Cargo.lock is 0.5.8, but Cargo.toml is 0.5.10. Should match match

Reply:
I must of deleted Cargo.lock at some point this is less changes